docs/BR.md

```diff
@@ -1,15 +1,12 @@
-**CA/Browser Forum**
-
+# Verso
```

Comment:
Do you mean the "Verso" line itself? No special meaning to it—I just stuck it in there because Pandoc will now generate a title page for us automatically. But since it does that dynamically, not having any sort of title information on the document looked weird, so I converted it into a 'Verso' section that will be the fly-leaf info for the document.

Reply:
Is "Verso" a special keyword for Pandoc? I guess I'm not familiar with it - could just be a lack of vocabulary, or if it was supposed to be a 'magic' phrase.

Reply:
It's nothing significant for Pandoc—just me being compulsive. :) It looked weird not having a title page in the Markdown, but then the PDF looked weird with a title page followed by a reiteration of the title page info. My solution was to keep the "extra" title page but label it 'Verso' (which, after [some reading](https://lts.library.cornell.edu/lts/rt/ref/ptsbk), doesn't seem quite correct--should probably go with "Colophon" or just "Document Information"). However, in looking over the resulting PDF, it looks like it's not doing what I expected, so I eliminated it in 376d176.